src/language/OpenShot/OpenShot_transitions.pot

```diff
@@ -360,8 +360,8 @@ msgstr ""
 msgid "Ray light right %s"
 msgstr ""
 
-#: /home/jonathan/apps/openshot-qt-git/src/transitions/extra/whirpool_3.jpg
-msgid "Whirpool %s"
+#: /home/jonathan/apps/openshot-qt-git/src/transitions/extra/whirlpool_3.jpg
+msgid "whirlpool %s"
```

Comment:
The name should be kept uppercase, in the translation master. (It'll actually be automatically capitalized when the file is regenerated by `src/language/generate_translations.py`, so it won't matter much anyway, but might as well keep it consistent so there's less churn in the translations.)

Reply:
Ohhhh. I think I had case sensitivity off on my editor. I'll be more careful with that